src/sbt-test/sbt-guardrail/scala-client-codegen-app/build.sbt

```
@@ -6,15 +6,20 @@ scalaVersion := "2.12.16"
 
 scalacOptions += "-Xexperimental"
 
-Compile / guardrailTasks := List(
-  ScalaClient(file("petstore.json"), pkg="com.example.clients.petstore", imports=List("_root_.support.PositiveLong")),
-  ScalaServer(file("petstore.json"), pkg="com.example.servers.petstore", imports=List("_root_.support.PositiveLong"))
-)
+Compile / guardrailTasks := GuardrailHelpers.createGuardrailTasks((Compile / sourceDirectory).value / "openapi") { openApiFile =>
+  List(
+    ScalaClient(openApiFile.file, pkg = openApiFile.pkg + ".client", imports = List("_root_.support.PositiveLong")),
+    ScalaServer(openApiFile.file, pkg = openApiFile.pkg + ".server", imports = List("_root_.support.PositiveLong"))
+  )
+}
+
+Test / guardrailTasks := (Test / guardrailDiscoveredOpenApiFiles).value.flatMap { openApiFile =>
+  List(
+    ScalaClient(openApiFile.file, pkg = openApiFile.pkg + ".client", imports = List("_root_.support.PositiveLong", "_root_.support.tests.PositiveLongInstances._")),
+    ScalaServer(openApiFile.file, pkg = openApiFile.pkg + ".server", imports = List("_root_.support.PositiveLong", "_root_.support.tests.PositiveLongInstances._"))
+  )
+}
```

perhaps
```suggestion
Test / guardrailTasks += ScalaClient.defaults(imports = List(
    "_root_.support.PositiveLong",
    "_root_.support.tests.PositiveLongInstances._"
  ))
Test / guardrailTasks ++= (Test / guardrailDiscoveredOpenApiFiles).value.flatMap { openApiFile =>
  List(
    ScalaClient(openApiFile.file, pkg = openApiFile.pkg + ".client"),
    ScalaServer(openApiFile.file, pkg = openApiFile.pkg + ".server")
  )
}
```

is a useful example for those attempting to reduce duplication